romancal/resample/resample_utils.py

```diff
@@ -163,6 +164,22 @@ def build_driz_weight(
     elif weight_type == "exptime":
         exptime = model.meta.exposure.exposure_time
         result = exptime * dqmask
+    elif weight_type == "ivsky":
+        if (
+            hasattr(model, "var_sky")
+            and model.var_sky is not None
+            and model.var_sky.shape == model.data.shape
+        ):
+            with np.errstate(divide="ignore", invalid="ignore"):
+                inv_sky_variance = model.var_sky**-1
+            inv_sky_variance[~np.isfinite(inv_sky_variance)] = 1
```

Comment:
Let's make the inv_sky_variance for these pixels 0 rather than 1.
inverse_variance = 0 -> no weight on these pixels
inverse_variance = 1 -> some amount of weight that depends on units...

Reply:
Done.